src/govuk/components/error-summary/error-summary.mjs

```diff
@@ -2,8 +2,18 @@ import '../../vendor/polyfills/Function/prototype/bind.mjs'
 import '../../vendor/polyfills/Event.mjs' // addEventListener
 import '../../vendor/polyfills/Element/prototype/closest.mjs'
 
-function ErrorSummary ($module) {
+import { mergeConfigs, normaliseDataset } from '../../common.mjs'
+function ErrorSummary ($module, config) {
```

Comment:
Whilst we're here, Can we add JSDoc blocks for at least the init function, as its signature has changed?